app/layout.tsx

```
@@ -6,12 +6,18 @@ import { SiteHeader } from "@/components/site-header";
 import { SiteFooter } from "@/components/site-footer";
 import { Analytics } from "@vercel/analytics/react";
 
+import { Analytics } from "@vercel/analytics/react";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Analytics'. Consider to delete it or rename it.

'Analytics' is defined here:


(lint/suspicious/noRedeclare)